.vale.ini

```diff
@@ -0,0 +1,6 @@
+StylesPath = /home/runner/work/uyuni-docs/uyuni-docs/styles/vale-styles
```

Comment:
I am not sure why we would need another subdirectory under `styles`? I would suggest to put all styles directly under `styles`, and move the `styles` directory to `.github` as [recommended in the vale action docs](https://github.com/errata-ai/vale-action#repository-structure).

Reply:
This was a test as styles were not being picked up by vale.